oidc-exchange.py

```diff
@@ -17,6 +17,12 @@
 
 {message}
 
+You're seeing this because the action wasn't given the inputs needed to
+perform password-based or token-based authentication. If you intended to
+perform one of those authentication methods instead of trusted
+publishing, then you should double-check your secret configuration and variable
+names.
```

Comment:
Suggestion by @joe733:
```suggestion
publishing (OIDC), then you should double-check your secret configuration
and variable names.
```

Reply:
FWIW, PyPI itself is trying to refer to this solely as "trusted publishing" -- OIDC is just the underlying technique, so public facing documentation should probably avoid mentioning it.

(PyPI's own docs aren't 100% consistent about that yet, unfortunately.)

Reply:
@woodruffw so this action's readme also needs to be updated, then...

Reply:
Yep; I can do that this evening (JST).